aiosmtpd/smtp.py

```diff
@@ -182,6 +187,21 @@ def eof_received(self):
             return False
         return super().eof_received()
 
+    def _reset_timeout(self):
+        if self._timeout_handle:
```

Comment:
Please use more explicit `is not None` check

Reply:
I originally used `is not None` but this led to code coverage issues.  Will add this back and see whether I can find a meaningful test case.